docs/diagrams/reset.md

```diff
@@ -0,0 +1,32 @@
+title etcdadm join https://member1:2379
+
+participant user
+participant remote repo
+participant etcdadm
+participant systemd
+database member1
+database member2
+
+user->etcdadm: reset
+
+group reconfigure etcd cluster
+etcdadm->member1: remove member2
+destroy member2
+end
+
+group uninstall artifacts
+etcdadm->etcdadm: delete etcd, etcdctl from install directory
+end
+
+group destroy certificates
+etcdadm->etcdadm: destroy CA certificate
```

Comment:
If `etcdadm reset` deletes the CA cert, then if you want to run `etcdadm join` again on the same machine (to the same cluster), you'll need to copy the CA cert over again, right?

Reply:
Yes, that's what currently happens.